src/Qir/Controller/Driver/QirDriverGenerator.cs

```diff
@@ -5,22 +5,48 @@
 using System.IO;
 using System.Threading.Tasks;
 using Microsoft.Quantum.Qir.Utility;
+using Microsoft.Quantum.QsCompiler;
 using Microsoft.Quantum.QsCompiler.BondSchemas.EntryPoint;
 
 namespace Microsoft.Quantum.Qir.Driver
 {
     public class QirDriverGenerator : IQirDriverGenerator
     {
+        private const string BytecodeFileName = "qir.bc";
+        private const string DriverFileName = "driver.cpp";
         private readonly ILogger logger;
 
         public QirDriverGenerator(ILogger logger)
         {
             this.logger = logger;
         }
 
-        public Task GenerateQirDriverCppAsync(DirectoryInfo sourceDirectory, EntryPointOperation entryPointOperation, ArraySegment<byte> bytecode)
+        public async Task GenerateQirDriverCppAsync(DirectoryInfo sourceDirectory, EntryPointOperation entryPointOperation, ArraySegment<byte> bytecode)
```

Comment:
nit: since this method creates both the C++ driver and the QIR bytecode files, consider renaming it to something like `GenerateQirExecutableSourceFiles`.